StandaloneMmPkg/Library/StandaloneMmMemLib/X86StandaloneMmMemLibInternal.c

```diff
@@ -6,28 +6,30 @@
   all MMRAM range via MM_ACCESS_PROTOCOL, including the range for firmware (like MM Core
   and MM driver) and/or specific dedicated hardware.
 
-  Copyright (c) 2015, Intel Corporation. All rights reserved.<BR>
+  Copyright (c) 2015 - 2024, Intel Corporation. All rights reserved.<BR>
   Copyright (c) 2016 - 2018, ARM Limited. All rights reserved.<BR>
   Copyright (c) Microsoft Corporation.
 
   SPDX-License-Identifier: BSD-2-Clause-Patent
 
 **/
+#include <StandaloneMmMemLibInternal.h>

```

Comment:
#include "StandaloneMmMemLibInternal.h"

Reply:
Thanks for the comments. Code has bee updated based on all the comments above